src/player/Player.php

```diff
@@ -2000,6 +2008,10 @@ public function emote(string $emoteId) : void{
 		}
 	}
 
+	public function consumeObject(Consumable $consumable, bool $forceEating = false) : bool{
+		return parent::consumeObject($consumable, $this->forceAllowEating() || $forceEating);
```

Comment:
Prefer conditions that consume less first, i.e. check the variable before the function.